Project.toml

```diff
@@ -49,35 +49,35 @@ UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
-AbstractFFTs = "0.5.0"
+AbstractFFTs = "0.5"
 Adapt = "1.0.1, 2"
 CUDA = "1.3"
-Combinatorics = "1.0.0"
+Combinatorics = "1.0"
 DataStructures = "0.17.9"
 FFTW = "1.2.0"
 FileIO = "1.2.2"
 Formatting = "0.4.0"
 ImageFiltering = "0.6.14"
 ImageMorphology = "0.2.8"
-Interpolations = "0.12.8"
+Interpolations = "0.12.8, 0.13"
 IterTools = "1.3.0"
 JLD2 = "0.1.11"
 KahanSummation = "0.1.0"
-Lazy = "0.13.2, 0.14, 0.15.0"
+Lazy = "0.13.2, 0.14, 0.15"
 Loess = "0.5.0"
 MacroTools = "0.5.4"
 Match = "1.1.0"
-Measurements = "2.0.0"
+Measurements = "2.0, 2.1, 2.3"
```

Comment:
I may not be understanding right, but isn't `2.0.0` and `2.0, 2.1, 2.3` the same compat bound, they both specify `[2.0.0, 3.0.0)` ?

Reply:
As far as I understand `"2"` specifies `[2.0.0, 3.0.0)` but `"2.0.0"` means `[2.0.0, 2.0.1)` because it is more specific. I believe they changed a few subtleties of the [**compat system**](https://julialang.github.io/Pkg.jl/dev/compatibility/) a while back, maybe they used to mean the same thing.

If you wanted to be as unrestrictive as possible, you should specify less "decimal places". So `"0.5"` which is `[0.5.0, 0.6.0)` instead of `"0.5.0"`. From my experience, CompatHelper always automatically suggests changes in the form of `"2.0"` -> `"2.0, 2.1"` when a new version of a package is released, which is why I just went with that although it may not be the ideal way.

Reply:
> As far as I understand "2" specifies [2.0.0, 3.0.0) but "2.0.0" means [2.0.0, 2.0.1)

Isn't that at odds with this line from your link:

```
PkgA = "^1.2.3" # [1.2.3, 2.0.0)
```

I do know there is a difference between between >1 and <1 versions though, for <1 versions minor only minor versions are assumed to be compatible with each other, thats why you get

```
PkgD = "^0.2.3" # [0.2.3, 0.3.0)
```

Reply:
But then you have to add the circumflex `^`, don't you?

Reply:
>But then you have to add the circumflex ^, don't you?

That's the default I think (also from your link):

> Caret specifiers are the default and hence 1.2.3 == ^1.2.3

Reply:
You're right, my bad. I thought that was only the default for versions beginning in "0" but I confused this with the equivalence of `~` and `^`. 

I just remember getting compatibility errors for **Optim.jl**, which is on 1.2 something now and **Roots.jl** which is on 1.0 something when I tried to download **CMBLensing.jl**.

Reply:
Yea, the Roots upper bound should indeed be upgraded now that there's 1.0, so thanks for that (I guess this is what CompatHelper will for you in the future right?) Anyway will tweak these last versions and merge shortly.

Reply:
Glad I could help.